src/api/v1/regions/community_businesses.ts

```diff
@@ -0,0 +1,38 @@
+import * as Hapi from 'hapi';
+import * as Boom from 'boom';
+import { pick, map } from 'ramda';
+import { CommunityBusinesses } from '../../../models';
+import { response } from '../schema/response';
+
+
+export default [
+  {
+    method: 'GET',
+    path: '/regions/{regionId}/community-businesses',
+    options: {
```

Comment:
Let's add some validation on the `regionId` parameter? There's a `id` schema knocking about somewhere (I think in schema/request).